pyvo/registry/regtap.py

```diff
@@ -895,6 +896,9 @@ def get_tables(self, table_limit=20):
         return res
 
 
+@deprecated("1.5", "ivoid2service does not work in the presence of"
+    " multiple capabilities.  Use"
+    " registry.search(ivoid=...)[0].get_service('capname') instead.")
```

Comment:
Maybe chop up the `message` arg to `message` and `alternative`
```suggestion
@deprecated("1.5", message="ivoid2service does not work in the presence of multiple capabilities",
            alternative=" registry.search(ivoid=...)[0].get_service('capname')")
```